src/Missings.jl

```diff
@@ -514,4 +514,32 @@ julia> emptymissing(first)([1], 2)
 """
 emptymissing(f) = (x, args...; kwargs...) -> isempty(x) ? missing : f(x, args...; kwargs...)
 
+# Variant of `isless` where `missing` is the smallest value
+"""
+    missingsmallest(f::Function)
+
+Creates a function of two arguments `x` and `y` that tests whether `x` is less
+than `y` such that `missing` is always less than the other argument. In other
```

Comment:
and otherwise `f` is used to perform the comparison.

Reply:
I feel like this is already covered in the last sentence "... `missing` is the smallest possible value,
and all other non-`missing` values are compared according to `f`."

Reply:
It would be good for the first sentence to give a complete and accurate description of what happens. "is less" is not really accurate here since it's just the result of `f`. The interpretation of that result as meaning "is less" only appears when you use this function in combination with `sort`.